src/components/Data/Documents/Views/TimeSeries.vue

```diff
@@ -1,28 +1,26 @@
 <template>
   <div class="col s12 TimeSeriesView">
-    <div class="row col s10">
-      <GChart
+    <div class="row chart-container">
+      <VueApexCharts
         v-if="customNumberFields.length"
-        type="LineChart"
-        :data="chart"
+        class="w-75"
+        type="line"
+        ref="Chart"
+        :series="series"
         :options="chartOptions"
-        :resize-debounce="1"
-        :settings="{ packages: ['corechart'], language: 'en' }"
       />
-      <div v-else class="row col s12">
-        No data to display
-      </div>
+      <div v-else class="row col s12">No data to display</div>
     </div>
-    <div class="col s2">
-      <div class="col s12 bordered">
+    <div class="row">
+      <div class="col s12">
         <span>Date</span>
         <autocomplete
           placeholder="Date field"
           :items="mappingDateArray"
           :value="customDateField || ''"
           :notify-change="false"
           @autocomplete::change="
-            item => {
+            (item) => {
```

Comment:
Nitpicking: why did you add the parenthesis? We only need them with at least 2 variables